pkg/deploy/deployment_postgres.go

```diff
@@ -122,4 +124,12 @@ func NewPostgresDeployment(cr *orgv1.CheCluster, chePostgresPassword string) *ap
 			},
 		},
 	}
+	if ! isOpenshift {
+		var runAsUser int64 = 26
```

Comment:
Why 26?

Reply:
To be consistent with upstream Che k8s helm charts: https://github.com/eclipse/che/blob/master/deploy/kubernetes/helm/che/custom-charts/che-postgres/templates/deployment.yaml#L26

Reply:
ok thanks and it looks consistent with the [upstream image user](https://github.com/sclorg/postgresql-container/blob/master/src/Dockerfile#L113)